tox.ini

```diff
@@ -0,0 +1,51 @@
+[tox]
+envlist = py37
+isolated_build = true
+skipsdist=True
+
+toxworkdir={toxinidir}/.tox
+
+[testenv]
+whitelist_externals =
+    sh
+    docker-compose
+commands_pre =
+    docker-compose up --detach mysql redis rabbitmq
+    pip install -r {toxinidir}/requirements/dev.txt
+    pip install -r {toxinidir}/requirements/common.txt
+    pip install -r {toxinidir}/requirements/docs.txt
+    {toxinidir}/manage.py check
+commands =
+    pip check
+    {toxinidir}/lints/queuelint.py
+    flake8 --show-source
+    mkdocs build
+    pre-commit run --all-files shellcheck
+    pre-commit run --all-files markdownlint
+    sh -c "SITE_URL=https://treeherder.dev TREEHERDER_DEBUG=False ./manage.py check --deploy --fail-level WARNING"
```

Comment:
@shubhank-saxena are these lines running in a docker container?  this will not work on Windows

Reply:
You can probably drop `sh -c` and it will still work.

Reply:
> Reverting `dev.txt` changes and removing the use of `sh` should be sufficient to land this. Thanks!

The issue of using sh -c is that tox wasn't handling env setting properly

Reply:
OK; handle the `dev.txt` and we can look into it later.